[*] Running ...
[*] Dataset loaded: E:\Projects\CO\reevo-main/problems/tsp_gls\dataset/train200_dataset.npy with 10 instances.
Traceback (most recent call last):
  File "E:\Projects\CO\reevo-main/problems/tsp_gls/eval.py", line 49, in <module>
    obj = solve(instance)
          ^^^^^^^^^^^^^^^
  File "E:\Projects\CO\reevo-main/problems/tsp_gls/eval.py", line 19, in solve
    heu = heuristics(inst.distmat.copy())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\Projects\CO\reevo-main\problems\tsp_gls\gpt.py", line 17, in heuristics_v2
    if i != j:
               
NameError: name 'direct_distance_penalty_weight' is not defined. Did you mean: 'direct_penalty_weight'?
